### ISSUE TITLE
Provide the capability to debug quarkus and qute ls

### ISSUE THREAD

**Issue by angelozerr** [redhat-developer/intellij-quarkus#769]
To improve LSP client for IJ we need to debug the MicroProfile LS or Qute LS. For instance we could add à breakpoint when the language server is initialized and see what happens on LSP client for IJ (to emulate à long start to initialize the language server).

Or we could set à breakpoint when diagnostic is published and update the IJ text editor to publish bad diagnostic which is unsynchronized with the document to publish diagnostic with bad offset.

In other words we could emulate corner usecase by setting breakpoint to stop the process from some opération and emuluate some slow operation.

I will work on this issue.
